scripts/create_expression_parser.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script produces the expression parser."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import fileinput
+import os
+import re
+import subprocess
+
+import python_utils
+
+from . import common
+from . import install_third_party_libs
+from . import setup
+
+
+def main():
+    """Produces the expression parser."""
+    setup.main()
```

Comment:
Explicitly pass empty args.

Reply:
Setup and setup_gae accepts no arguments and hence its main accepts no argv